app/element-finder-sync.ts

```diff
@@ -146,8 +146,10 @@ export class ElementFinderSync {
       try {
         exec(this.element.click());
       } catch (e) {
-        if (autoRetryClick && /Other element would receive the click/.test(e.message) && new Date().getTime() - startTime < implicitWaitMs) {
-          console.log('(Protractor-sync): Element (' + this.getSelectionPath() + ') was covered, retrying click.');
+        //Unfortunately there's no way to detect this except for checking the string contents.
+        const isClickError = /Element is not clickable at point/.test(e.message) || /Other element would receive the click/.test(e.message);
```

Comment:
The variable name is generic b/c it's representing two possible sources of click errors. I'm wondering if you might have missed the "||" b/c the variable name suggested just covers the first half. If that's not the case I'm sure we can find a variable name we both like :)

Reply:
Thanks, got it. To my mind 'isClickError' is still too general because it sounds like it could encompass other types of click errors (e.g. WebDriver choked trying to perform the click, element went stale, this VM is configured such that it doesn't allow clicking, etc.) But I see how my suggestion sounds too specific. :) Might as well keep it the way it is instead of simply introducing the opposite problem!